database/index.js

```diff
@@ -0,0 +1,63 @@
+const { Client } = require('pg');
+const Promise = require('bluebird');
+
+const client = new Client({
+  user: process.env.LENA_ORDERS_DB_USER || 'lena',
+  host: process.env.LENA_ORDERS_DB_HOST || 'localhost',
+  database: process.env.LENA_ORDERS_DB_data || 'transactions',
```

Comment:
LENA_ORDERS_DB_data should be all capitalized

Reply:
can't believe I missed that!